test/inductor/test_codecache.py

```diff
@@ -287,6 +291,50 @@ def fn2(x):
         self.assertEqual(counters["inductor"]["fxgraph_cache_miss"], 2)
         self.assertEqual(counters["inductor"]["fxgraph_cache_hit"], 0)
 
+    @requires_gpu()
+    @requires_triton()
+    @config.patch({"fx_graph_cache": True})
+    def test_higher_order_op_bypass(self):
+        """
+        Verify that we bypass the cache when we have higher order ops.
+        """
+
+        @triton.jit
+        def add_kernel(
```

Comment:
no need to redefine this, import it from torch/testing/_internal/triton_utils.py